float64 konumX
float64 konumY